src/main/java/com/ibm/appscan/jenkins/plugin/auth/ASoCCredentials.java

```diff
@@ -12,7 +12,7 @@ public class ASoCCredentials extends com.hcl.appscan.jenkins.plugin.auth.ASoCCre
 
 	private static final long serialVersionUID = 1L;
 
-	public ASoCCredentials(CredentialsScope scope, String id, String description, String username, String password) {
-		super(scope, id, description, username, password);
+	public ASoCCredentials(CredentialsScope scope, String id, String description, String username, String password, Boolean certificates) {
```

Comment:
As mentioned previously, "certificates" is not a good name for this variable. Something like "acceptssl" or "acceptInvalidCerts" would be more descriptive.

Reply:
Now, I am using "acceptInvalidCerts".